packages/workspace/src/browser/workspace-uri-contribution.ts

```diff
@@ -86,13 +106,11 @@ export class WorkspaceUriLabelProviderContribution extends DefaultUriLabelProvid
      */
     getLongName(element: URI | FileStat): string {
         const uri = this.getUri(element);
-
-        if (this.wsRoot) {
-            const relativeUri = this.wsRoot.relative(uri);
+        if (this.baseUri) {
+            const relativeUri = this.baseUri.relative(uri);
```

Comment:
btw it looks bogus, we should workspace root which is the parent, not first random

Reply:
no it is not the `first random` :) 

the code uses the first root when there is only one root folder in the workspace.